src/lib/utils/options/options.ts

```diff
@@ -72,6 +74,10 @@ export class Options extends ChildableComponent<Application, OptionsComponent> {
         this.trigger(event);
         this.setValues(event.data, '', event.addError.bind(event));
 
+        if (event.compilerOptions) {
+            _.assign(this.compilerOptions, event.compilerOptions);
```

Comment:
It can be used, but currently `event.data` is more like `event.rawOptions` so whatever additional data is added it should not be in conflict with any of the options and should be deleted before parsing them.
If the following is OK I'll make the changes:
1. in `readers/tsconfig.ts` - add `parsedCompilerOptions: CompilerOptions` to the `event.data`
2. in `options/options.ts ` - delete `parsedCompilerOptions` from `event.data` before parsing